pkgs/development/python-modules/orange3/default.nix

```
@@ -0,0 +1,127 @@
+{ lib
+, buildPythonPackage
+, copyDesktopItems
+, fetchurl
+, makeDesktopItem
+, fetchFromGitHub
+, nix-update-script
+, pythonRelaxDepsHook
+, baycomp
+, bottleneck
+, chardet
+, cython
+, httpx
+, joblib
+, keyring
+, keyrings-alt
+, matplotlib
+, numpy
+, openpyxl
+, opentsne
+, orange-canvas-core
+, orange-widget-base
+, pandas
+, pyqtgraph
+, pyqtwebengine
+, python-louvain
+, pyyaml
+, qt5
+, qtconsole
+, requests
+, scikit-learn
+, scipy
+, serverfiles
+, xlrd
+, xlsxwriter
+}:
+
+buildPythonPackage rec {
+  pname = "Orange3";
+  version = "3.35.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "biolab";
+    repo = "orange3";
+    rev = "refs/tags/${version}";
+    hash = "sha256-dj/Z4uOjA4nidd45dnHZDyHZP6Fy/MGC8asqOPV7U7A=";
+  };
+
+  pythonRelaxDeps = [
+    "scikit-learn"
+    "pyqtgraph" # TODO: remove after bump with a version greater than 0.13.1
+  ];
+
+  nativeBuildInputs = [
+    copyDesktopItems
+    cython
+    pythonRelaxDepsHook
+    qt5.wrapQtAppsHook
+  ];
+
+  enableParallelBuilding = true;
+
+  propagatedBuildInputs = [
+    numpy
+    scipy
+    chardet
+    openpyxl
+    opentsne
+    qtconsole
+    bottleneck
+    matplotlib
+    joblib
+    requests
+    keyring
+    scikit-learn
+    pandas
+    pyqtwebengine
+    serverfiles
+    orange-canvas-core
+    python-louvain
+    xlrd
+    xlsxwriter
+    httpx
+    pyqtgraph
+    orange-widget-base
+    keyrings-alt
+    pyyaml
+    baycomp
+  ];
+
+  # FIXME: ImportError: cannot import name '_variable' from partially initialized module 'Orange.data' (most likely due to a circular import) (/build/source/Orange/data/__init__.py)
+  dontCheck = true;
+  pythonImportsCheck = [ "Orange" "Orange.data._variable" ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "orange";
+      exec = "orange-canvas";
+      desktopName = "Orange Data Mining";
+      genericName = "Data Mining Suite";
+      comment = "Explore, analyze, and visualize your data";
+      icon = "orange-canvas";
+      mimeTypes = [ "application/x-extension-ows" ];
+      categories = [ "Science" "Education" "ArtificialIntelligence" "DataVisualization" "NumericalAnalysis" "Qt" ];
+      keywords = [ "Machine Learning" "Scientific Visualization" "Statistical Analysis" ];
+    })
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/orange-canvas \
+      "${"$"}{qtWrapperArgs[@]}"
+    mkdir -p $out/share/icons/hicolor/{256x256,48x48}/apps
+    cp distribute/icon-256.png $out/share/icons/hicolor/256x256/apps/orange-canvas.png
+    cp distribute/icon-48.png $out/share/icons/hicolor/48x48/apps/orange-canvas.png
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    mainProgram = "orange-canvas";
+    description = "Data mining and visualization toolbox for novice and expert alike";
+    homepage = "https://orangedatamining.com/";
+    license = [ lib.licenses.gpl3 ];
```

```suggestion
    license = lib.licenses.gpl3Plus;
```
https://github.com/biolab/orange3/blob/ff152a9daffe9d68963f5a6f2003e96bdcb0d45d/setup.py#L73